backend-db/modules/db/couchdb/index.js

```diff
@@ -900,6 +953,63 @@ const layer = {
         }
         return false;
     },
+
+    //========================================================
+    // TEAMS
+    //========================================================
+    getTeam: id => adapter.getOneEntityById('Team', '_design/byTeamId/_view/view', id),
+    getAllTeams: () => adapter.getEntities('Team', '_design/all/_view/view', {sorted: true}),
+    getAllTeamsByCountry: async (country, search) => {
+      const teams = await adapter.getEntities(
+        'Team',
+        '_design/all/_view/view',
+        {
+          sorted: true,
+        }
+      );
+      const filteredTeams = search ? teams.filter(team =>
+        team.name && team.name.toUpperCase().indexOf(search.toUpperCase()) > -1 ||
+        team.teamDescription && team.teamDescription.toUpperCase().indexOf(search.toUpperCase()) > -1 ||
+        COUNTRY_LIST[team.CC] && COUNTRY_LIST[team.CC].toUpperCase().indexOf(search.toUpperCase()) > -1
+      ) : teams;
+      const sortedTeamsByCountry = _.sortBy(filteredTeams, team => team.CC !== country);
+      return sortedTeamsByCountry
+    },
+    getCountTeamsTrashpoints: () => adapter.getEntities('Team', '_design/all/_view/view', {sorted: false}),
+    getRawTeamDoc: id => adapter.getOneRawDocById('Team', '_design/byTeamId/_view/view', id),
+    createTeam: async (id, who, create) => {
+      await adapter.createDocument('Team', id, create, {
+        createdAt: util.time.getNowUTC(),
+        createdBy: who || undefined,
+      });
+      return await layer.getTeam(id);
+    },
+    seedTeams: async (metadata) => {
+      console.log('METADATA', metadata);
+      const ret = await layer.getAllTeams();
+      console.log('RET', ret);
+      if (!Array.isArray(ret)) {
+        return false;
+      }
+      console.log(' IT IS HERERERERERE');
+      const existingTeams = ret.reduce((prev, team) => {
+        prev[team.id] = team;
+        return prev;
+      }, {});
+      const neededParams = ['name', 'teamDescription', 'CC'];
+      _.each(metadata, async (team, key) => {
+        const existedTeam = existingTeams[team.id];
+        if (!existedTeam && (!team.CC || team.CC && CCs.includes(team.CC))) {
```

Comment:
`!A || A ... === !A || ...`

Isn't this line missing another pair of parens or the above simplification? If `!teams.CC` is truthy, it will also execute the `includes` check, which will fail, so the team wouldn't get created.
```
!team.CC || (team.CC && CCs.includes(team.CC))
!team.CC || CCs.includes(team.CC)
```

But, shouldn't `CCs.includes(team.CC)` be a panicky check somewhere separate — in case it turns out not to be true, creating a team can create bad data, causing problems later down the road.

Reply:
Users should not be able to create new teams as we don't have all the related functions designed and developed.

Reply:
This is not about users creating them, seeding is the import of the static data.